### PR TITLE
crio: add userns job/configuration for nodes that will run userns jobs

### PR DESCRIPTION
**Pull request by haircommander**
as well as generate the crio ignition files with privilege or else it fails with selinux enabled

### CODE REVIEW

**Review by rata — APPROVED** [review]
@haircommander LGTM, thanks A LOT for this PR!

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -1082,6 +1082,56 @@ presubmits:
...
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240223-1ded72f317-master
+        command:
+          - runner.sh
+          - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --deployment=node
+        - --env=KUBE_SSH_USER=core
+        - --gcp-zone=us-west1-b
+        - '--node-test-args=--service-feature-gates="UserNamespacesSupport=true,ProcMountType=true" --feature-gates="UserNamespacesSupport=true,ProcMountType=true" --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
```
Comment (kannon92) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1119]:
```suggestion
        - '--node-test-args=--service-feature-gates="UserNamespacesSupport=true,ProcMountType=true" --feature-gates="UserNamespacesSupport=true,ProcMountType=true" --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service” --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
```

I don’t think extra logs is part of the kubelet flags. @rphillips helped spot this in my PR earlier today.

Comment (rata) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1119]: Oh, good catch, we need to close `"` there. But it seems your suggestion is using a non-ascii `"`, that probbly won't work.

But great catch :)

**Comment by rata** [comment]
Thanks! All green now! :)

**Comment by SergeyKanzhelev** [comment]
can you please rebase so I can approve?

**Comment by rata** [comment]
@haircommander friendly ping? :)

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

**Pull request merged**